packages/Testimonial/Testimonial.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Box from '@tds/core-box'
+import Card from '@tds/core-card'
+import Image from '@tds/core-image'
+import Paragraph from '@tds/core-paragraph'
+import safeRest from '@tds/shared-safe-rest'
+
+import styles from './Testimonial.scss'
+
+/**
+ * @version ./package.json
+ */
+/** Testimonial component for displaying testimonial in a standalone, pre-styled component so all testimonials look consistent across the website */
+const Testimonial = ({ testimonial, imgSrc, imgAlt, title, additionalInfo, ...rest }) => {
+  return (
+    <Card {...safeRest(rest)}>
+      <Box between={3}>
+        <Paragraph>&quot;{testimonial}&quot;</Paragraph>
+        <Box inline={3} dangerouslyAddClassName={styles['testimonial-author']}>
+          {imgSrc && <Image src={imgSrc} alt={imgAlt} width={60} height={60} rounded="circle" />}
+          <Box horizontal={imgSrc ? 3 : 0}>
+            <Paragraph bold>{title}</Paragraph>
+            <Paragraph>{additionalInfo}</Paragraph>
+          </Box>
+        </Box>
+      </Box>
+    </Card>
+  )
+}
+
+Testimonial.propTypes = {
+  /** Testimonial main text */
+  testimonial: PropTypes.string.isRequired,
```

Comment:
Not 100% sure about allowing `PropTypes.element` or `PropTypes.node`. Wouldn't that allow too much of a flexibility to it like passing an entire React Component in there for example?

Reply:
I guess it might be good to ask design, if the `testimonial` in this layout:
* will _**never**_ have multiple paragraphs?
* will _**never**_ have links?
* will _**never**_ have superscript (legal text annotation)?
* will _**never**_ have formatted text (bold, italic, etc.)?

If the component would *_ever_* need to include *_any_* of those elements, then we should allow `PropTypes.node` as well. There's a risk that if we make it too strict / rigid, then outcome teams won't adopt the component.

I support starting out with `PropTypes.string` for now, and then opening it up to also accept `PropTypes.node` etc. later, since that would be a non-breaking change, so it could just be a patch release.

But, in any case, this is just a comment / open question. Not blocking the PR for this at all. 😃